extension/app/scripts.babel/pwa-check.js

```diff
@@ -78,6 +81,7 @@ function createResultsHTML(results) {
     resultsHTML +=
       `<li class="${groupClass}">
         <span class="group-name">${group}</span>
+        <span class="group-score">(${score}/${total})</span>
```

Comment:
again eventually this "score" concept should be the responsibility of a presenter layer, but for now :+1: